packages/fxa-content-server/app/scripts/templates/sign_in_password.mustache

```diff
@@ -97,4 +106,14 @@
 
 <!-- New L10N Strings
 {{#unsafeTranslate}}Enter your password <span class="card-subheader" id="subheader">for your Mozilla account</span>{{/unsafeTranslate}}
+{{#unsafeTranslate}}
+  By proceeding, you agree to:<br />
+  Pocket’s <a class="link-grey" id="pocket-tos" href="https://getpocket.com/tos/">Terms of Service</a> and <a id="pocket-pp" class="link-grey" href="https://getpocket.com/privacy/">Privacy Notice</a><br />
+  Mozilla Accounts <a class="link-grey" id="fxa-tos" href="/legal/terms">Terms of Service</a> and <a class="link-grey" id="fxa-pp" href="/legal/privacy">Privacy Notice</a>
```

Comment:
Was this not already translated in another file?

If so, I think we can remove these. If not, I think we only need the new string commented out once (e.g. just in one of these files instead of all of them).

Reply:
I think the pocket terms was missed when we added new strings for the rebranding. I didn't realize we only needed to add the new strings once though - @bcolsson can you confirm that this is correct, terms extracted with gettext only need to be added once even if they're used in multiple places?

Reply:
The way gettext is setup for this project (and most gettext projects) is that the source string is also the message ID (key) for the translation. So as long as the source strings are exactly the same they will be combined into the same translation entity.

[This](https://github.com/mozilla/fxa-content-server-l10n/blob/2312703ea66faba88b5a00909a74881f9b02c3ae/locale/templates/LC_MESSAGES/client.pot#L1285) is an example of a string being used across multiple files.

So in the sense that we're adding a string to trigger translation before it starts being used, we likely only need to add it to one file to start the translation process.